src/Cafe/OS/libs/gx2/GX2.cpp

```diff
@@ -343,7 +345,7 @@ void _GX2SubmitToTCL()
 	// update last submitted CB timestamp
 	uint64 commandBufferTimestamp = Latte_GetTime();
 	LatteGPUState.lastSubmittedCommandBufferTimestamp.store(commandBufferTimestamp);
-	gx2Log_printf("Submitting GX2 command buffer with timestamp %016I64x", commandBufferTimestamp);
+	gx2Log_printf("Submitting GX2 command buffer with timestamp %016" SCNx64, commandBufferTimestamp);
```

Comment:
To be honest, I didn't even know that there were separate types for printing and scanning, but let me fix this. As for using `cemuLog_log` instead of `gx2Log_printf`wouldn't it be odd to mix both? So IMO either update all or nothing.

Reply:
gx2Log_printf is deprecated, but yeah. I guess we'll remove this in a full passthrough.